scannerc.go

```diff
@@ -1996,14 +1997,15 @@ func yaml_parser_scan_tag_uri(parser *yaml_parser_t, directive bool, head []byte
 			}
 		} else {
 			s = read(parser, s)
+			length++
```

Comment:
This doesn't work when the tag name consists entirely of URL-escaped characters.